src/expr/node.h

```diff
@@ -37,13 +37,14 @@
 #include "base/cvc4_assert.h"
 #include "base/exception.h"
 #include "base/output.h"
-#include "expr/type.h"
-#include "expr/kind.h"
-#include "expr/metakind.h"
 #include "expr/expr.h"
 #include "expr/expr_iomanip.h"
+#include "expr/kind.h"
```

Comment:
I am going to be annoying and ask this be in separate commit. node.h and these are mutually recursive and hence a bit flaky to re-ordering. This should be fine and should be done. Just I would like to be able to roll this back if we find it stops compiling on some obscure path.

Reply:
Good point, reverted the reordering.